### PR TITLE
CIV-6282 Judicial Decision doc preview

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-6282

### Change description ###
Judges and Legal advisors have the ability to review the PDF that will be created following their decision. Additional screens have been implemented to preview the document before CYA screen. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/tests/api_unspec_claim_tests/api_1v1_adjourn_vacate_ga_test.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ let civilCaseReference, gaCaseReference;
 
 Feature('GA 1v1 Make Adjourn Vacate API tests @ignore-api-tests');
 
-Scenario('AC 4 - 15 Days with consent', async ({api}) => {
```
Comment (karthick-mohanasundaram-HMCTS) [inline e2e/tests/api_unspec_claim_tests/api_1v1_adjourn_vacate_ga_test.js:9]: revert

File: ga-ccd-definition/CaseEventToComplexTypes/JudicialDecisionGAspec.json [hunk 2]
```diff
@@ -58,18 +58,29 @@
     "ID": "GAJudicialMakeADecisionScreen",
     "CaseEventID": "MAKE_DECISION",
     "CaseFieldID": "judicialDecisionMakeOrder",
-    "ListElementCode": "judgeApproveEditOptionDate",
+    "ListElementCode": "judgeApproveEditOptionDoc",
     "EventElementLabel": " ",
     "FieldDisplayOrder": 6,
-    "DisplayContext": "MANDATORY"
+    "DisplayContext": "MANDATORY",
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/CaseEventToComplexTypes/JudicialDecisionGAspec.json:64]: broken, work in progress

File: ga-ccd-definition/CaseEventToComplexTypes/JudicialDecisionGAspec.json [hunk 3]
```diff
@@ -58,18 +58,29 @@
...
   {
     "ID": "GAJudicialMakeADecisionScreen",
     "CaseEventID": "MAKE_DECISION",
     "CaseFieldID": "judicialDecisionMakeOrder",
-    "ListElementCode": "dismissalOrderText",
+    "ListElementCode": "judgeApproveEditOptionDate",
     "EventElementLabel": " ",
     "FieldDisplayOrder": 7,
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "makeAnOrder=\"APPROVE_OR_EDIT\" AND displayjudgeApproveEditOptionDate=\"Yes\""
```
Comment (karthick-mohanasundaram-HMCTS) [inline ga-ccd-definition/CaseEventToComplexTypes/JudicialDecisionGAspec.json:75]: broken, work in progress

**Comment by karthick-mohanasundaram-HMCTS** [comment]
- [ ] Don't merge as new screen title page is yet to confirm with Steve by Alex

**Review by vasudevganesanhmcts — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by pliao-hmcts — APPROVED** [review]

**Pull request merged**